.idea/compiler.xml

```diff
@@ -7,7 +7,14 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="Blogg" />
+        <module name="Bloggy" />
       </profile>
     </annotationProcessing>
   </component>
+  <component name="JavacSettings">
+    <option name="ADDITIONAL_OPTIONS_OVERRIDE">
+      <module name="Blogg" options="-parameters" />
+      <module name="Bloggy" options="-parameters" />
+    </option>
+  </component>
 </project>
```

Comment:
remove this file (add this please to .gitignore)

Reply:
Done, catalogue removed.
`git rm --cached idea/\*`
https://superuser.com/questions/1397199/how-to-ignore-a-tracked-file-in-git-without-deleting-it

Reply:
good job!